python/paddle/incubate/autograd/utils.py

```diff
@@ -218,6 +227,37 @@ def get_output_var_list(op):
         ]
 
 
+def get_output_vars_from_comosite(op):
```

Comment:
bad name, maybe call it `rearrange_output_for_composite`

Reply:
done